src/Action/Dummy.hs

```diff
@@ -0,0 +1,53 @@
+{-# LANGUAGE GeneralizedNewtypeDeriving #-}
+{-# LANGUAGE MultiParamTypeClasses      #-}
+{-# LANGUAGE OverloadedStrings     #-}
+module Action.Dummy
+    ( DummyT(DummyT,unDummyT), Dummy
+    , runDummyT, runDummy
+    , notImplemented
+    ) where
+
+import Control.Monad.Except (runExceptT)
+
+import Action
+import Frontend.Prelude
+
+newtype DummyT m a = DummyT { unDummyT :: ExceptT ActionExcept m a }
+    deriving (Functor, Applicative, Monad, MonadError ActionExcept)
+
+type Dummy = DummyT Identity
+
+runDummyT :: DummyT m a -> m (Either ActionExcept a)
+runDummyT = runExceptT . unDummyT
+
+runDummy :: Dummy a -> Either ActionExcept a
+runDummy = runIdentity . runDummyT
+
+notImplemented :: Monad m => String -> String -> DummyT m a
+notImplemented meth _class = throwError500 $ unlines
```

Comment:
Leading `_` means "ghc: don't wanr about unused bind".  We've done this before, and lens does it, but I'd prefer if we didn't.  `cl`?